scripts/bootstrapCatalogWithSam.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Potentially stupid question: Why write this script in bash? It seems like we have a lot of intermediate variables being stored in env, which can both leave data around and override local env variables. If we use a scripting language like python, we get more power to use things like local variables and arg parsing libraries.

We could also potentially get some safeguards using the `gcloud-firecloud` library https://github.com/broadinstitute/firecloud-develop/tree/dev/run-context/live/scripts/migrations/gcloud_firecloud (although I don't know if it was something that has been opened up to general use yet, CI was working on it and were discussing that it could be useful for a lot of teams' scripting)

Reply:
this just exists as something reviewable and a record for what was done, I only need to run it one time and we're done with it. I followed the pattern in TDR
https://github.com/DataBiosphere/jade-data-repo/blob/develop/datarepoBootstrap.sh